.github/workflows/test.yml

```diff
@@ -0,0 +1,103 @@
+name: Test Variants
+
+on: push
+
+env:
+  COMPOSE_FILE: docker-compose.test.yml
+  APP_NAME: rails_templates
+  DOCKER_IMAGE: ${{ github.repository }}
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
+  DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
+  RUBY_VERSION: 2.7.1
+  NODE_VERSION: 12
+
+jobs:
+  test:
+    name: Test ${{ matrix.variant }} variant
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        variant: [web, api]
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+
+      - uses: nimblehq/branch-tag-action@v1.1
+
+      - uses: ruby/setup-ruby@v1.45.0
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Gems cache
+        uses: actions/cache@v2
+        with:
+          path: /home/runner/.rubies/ruby-${{ env.RUBY_VERSION }}
+          key: ${{ runner.os }}-gems-${{ env.RUBY_VERSION }}-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-${{ env.RUBY_VERSION }}-
+
+      - name: Create app folder
+        if: ${{ matrix.variant == 'web' }}
```

Comment:
@hoangmirs Could you explain why we don't need to run this for API variant @hoangmirs

Reply:
I think I made this by mistake, Removed it